python/tvm/relay/expr.py

```diff
@@ -0,0 +1,136 @@
+# pylint: disable=no-else-return, unidiomatic-typecheck, invalid-name
+"""The expression nodes of Relay."""
+from typing import List
+import tvm
+from .base import Span, NodeBase, register_relay_node
+from .type import Type, TypeParam
+from ._ir_pass import _get_checked_type
+from . import _make
+
+
+class Expr(NodeBase):
+    """The base type for all Relay exprressions."""
+
+    def checked_type(self):
+        return _get_checked_type(self)
+
+    def __call__(self, *args):
```

Comment:
since call of subfunction is rare, I am not sure if we want this now, instead of allow user to simply do relay.Call(var, args)